content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,30 +5,65 @@ weight = 20
 
 +++
 
-## What is Kubeflow?
+Kubeflow is the ecosystem of various applications created to address each stage in the AI/ML lifecycle,
+from exploration to training and serving. You can deploy Kubeflow components as standalone applications
+or deploy Kubeflow as an end-to-end AI/ML platform. Anywhere you are running Kubernetes,
+you should be able to run Kubeflow.
 
-Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it provides components for each stage in the ML lifecycle, from exploration through to training and deployment.
-Operators can choose what is best for their users, there is no requirement to deploy every component.
+There are a few ways to install Kubeflow:
 
-Learn more about Kubeflow in the [Introduction](/docs/started/introduction/) and
-[Architecture](/docs/started/architecture/) pages.
+- [**Install Kubeflow Components Standalone**](#install-kubeflow-components-standalone)
+- [**Install Kubeflow Platform from Raw Manifests**](#install-kubeflow-platform-from-raw-manifests)
+- [**Install Kubeflow Platform from Packaged Distributions**](#install-kubeflow-platform-from-packages-distributions)
 
-## How to install Kubeflow?
+## Install Kubeflow Components Standalone
 
-Anywhere you are running Kubernetes, you should be able to run Kubeflow.
-There are two primary ways to install Kubeflow:
+Kubeflow components can be deployed as standalone applications. You can integrate those components
+to your existing AI/ML platform. For example, for Model Training you can install Training Operator
+or for Model Serving you can install KServe.
 
-1. [**Packaged Distributions**](#packaged-distributions-of-kubeflow)
-1. [**Raw Manifests**](#raw-kubeflow-manifests) <sup>(advanced users)</sup>
+Follow the appropriate guides to install required Kubeflow components in standalone mode:
 
-<a id="packaged-distributions"></a>
-<a id="install-a-packaged-kubeflow-distribution"></a>
+- Install [Kubeflow Pipelines](/docs/components/pipelines/v2/installation/quickstart/)
+- Install [Kubeflow Training Operator](/docs/components/training/installation/#installing-training-operator)
+- Install [Kubeflow MPI Operator](/docs/components/training/user-guides/mpi/#installation)
+- Install [Kubeflow Katib] (TODO: Add link after #3723)
```

Comment:
I'm not sure about adding "TODO" here. Maybe "Not yet available", or something like that would be better. Also, add to https://github.com/kubeflow/katib/issues/2314 a task to update the link here.

Reply:
I want to merge this PR after Katib and Model registry docs will be updated: https://github.com/kubeflow/website/pull/3723, https://github.com/kubeflow/website/pull/3698.
So I can add the appropriate links here.